### PR TITLE
Windows June 3rd, 2015 release blog post

### PR DESCRIPTION
**Pull request by nikhilkh**

### CODE REVIEW

**Comment by nikhilkh** [comment]
When docs update happens, I need to insert a link for the Windows docs.

**Comment by stevengill** [comment]
I updated the edge documentation! Do you seem them in there?

**Comment by nikhilkh** [comment]
Thanks, Steve! I did not realize that you did. I have updated the blog post with the link now.

**Comment by purplecabbage** [comment]
LGTM!

File: 2015-06-03-windows-release.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+We are happy to announce that Cordova Windows 4.0.0 has been released!
+
+#Key features
+The release adds the following key features:
+* The default Windows target version is now 8.1. Windows 8.0 support is deprecated.
```
Comment (axemclion) [inline 2015-06-03-windows-release.md]: Should we talk about the deprecation time, and what happens in 6 months ?

**Comment by axemclion** [comment]
Minor comments 
- I think we should also talk about what happens when Windows 10 finally ships, if there are updates to the preview, etc
- Is Windows 10 production ready - I guess not, right, given that the platform is still in preview ? 
- Is there any plugin impact due to this ? Will all plugins work 
- Are we changing the `windows` and `windows8` platform keywords ? One of them is not deprecated, right ?

File: 2015-06-03-windows-release.md (hunk 1 above)
Comment (nikhilkh) [inline 2015-06-03-windows-release.md]: Good point. I have fixed this in the next iteration!

**Comment by nikhilkh** [comment]
* Added some details about known issues and gotchas and a note about future updates with provide more support as Windows RTM happens. It's likely we won't break anything, only add more features.
* Yes, plugins should all work with Windows 10 though our docs go into more details about different capabilities which work or not in different modes - ms-appx/ms-appx-web.
* I'm not sure what the `windows8` platform keyword is used for? We should ideally depecrate it.

**Comment by purplecabbage** [comment]
+1 to deprecating the `windows8` platform keyword

**Comment by nikhilkh** [comment]
Done - deprecation notice for `windows8` added.

**Pull request merged**